mkdocs.yml

```diff
@@ -71,9 +71,10 @@ nav:
 - Course Administration:
   - Setup Course Properties: 'courseadministration/courseproperties.md'
   - Grant Permissions: 'courseadministration/permissions.md'
-  - Add Content: 
+  - Add Content:
     - Modules: 'courseadministration/addcontent/modules.md'
-    - Learning Units: 
+    - Quiz Import: 'courseadministration/addcontent/quiz_import.md'
```

Comment:
I would propose to move this to "Learning Units" as well, maybe just below "Quiz"?